docs/maui/native-library-interop/get-started.md

```diff
@@ -106,17 +113,16 @@ On the native side, make updates in _template/android/native/app/src/main/java/c
 
 Back on the .NET side, we are now ready to interop with the native library:
 1. Run `dotnet build` from _template/android/NewBinding.Android.Binding_ to test everything is plugged in correctly and good to go. (Note: This step will require that you have JDK 17 installed)
```

Comment:
Update gradle to prevent the following warning from showing:

```
You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy
  NewBinding.Android.Binding succeeded with 1 warning(s) (19.1s) → bin/Debug/net9.0-android/NewBinding.Android.Binding.dll
    GRADLE : warning : SDK processing. This version only understands SDK XML versions up to 3 but an SDK XML file of version 4 was encountered. This can happen if you use versions of Android Studio and the command-line tools that were released at different times.
    ```

Reply:
I've updated the gradle versions here https://github.com/CommunityToolkit/Maui.NativeLibraryInterop/pull/55/commits/af5e06984eb6e9f86e75763b6a35cc4eb46a462d to fix this warning